src/Tests/dotnet-publish.Tests/GivenDotnetPublishPublishesProjects.cs

```diff
@@ -340,5 +343,31 @@ public void DotnetPublishAllowsPublishOutputDir()
                 .Should()
                 .Pass();
         }
+
+
+        [Fact]
+        public void A_PublishRelease_property_does_not_override_other_command_configuration()
+        {
+            var helloWorldAsset = _testAssetsManager
+               .CopyTestAsset("HelloWorld", "PackHelloWorld")
+               .WithSource();
+
+            System.IO.File.WriteAllText(helloWorldAsset.Path + "/Directory.Build.props", "<Project><PropertyGroup><PublishRelease>true</PublishRelease></PropertyGroup></Project>");
+
+            new BuildCommand(helloWorldAsset)
+               .Execute()
+               .Should()
+               .Pass();
+
+            // Another command, which should not be affected by PublishRelease
+            var packCommand = new DotnetPackCommand(Log, helloWorldAsset.TestRoot);
+
+            packCommand
+                .Execute()
+                .Should()
+                .NotHaveStdOutContaining("Release")
```

Comment:
Same here, I'd suggest checking that the `bin\Release` folder doesn't exist (and the `bin\Debug` one does).